lib/api/LogManagerImpl.hpp

```diff
@@ -297,8 +297,9 @@ namespace MAT_NS_BEGIN
         static size_t GetDeadLoggerCount();
 
         virtual void SetDataInspector(const std::shared_ptr<IDataInspector>& dataInspector) override;
-
+        virtual void ClearDataInspectors() override;
         virtual std::shared_ptr<IDataInspector> GetDataInspector() noexcept override;
+        virtual std::shared_ptr<IDataInspector> GetDataInspector(const std::string& uniqueIdentifier) noexcept override;
```

Comment:
Since we can get specific `IDataInspector` based on the identifier, I'd suggest allowing for removing specific inspector based on that as well.

Reply:
added